phantomgames/Home/CustomCells/CustomHomeTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CustomHomeTableViewCell.swift
+//  phantomgames
+//
+//  Created by Aphiwe Shozi on 2024/05/14.
+//
+
+import UIKit
+import SDWebImage
```

Comment:
Why is this still being used??

Reply:
I have not been using it in my newer features and I am currently prioritising features that need to be developed not those that have been completed. I will remove it but its not a priority at the moment